package.json

```diff
@@ -72,6 +72,7 @@
     "html-escaper": "^3.0.3",
     "jest": "^26.6.3",
     "jest-cli": "^28.0.2",
+    "jsonlint-mod": "^1.7.6",
```

Comment:
No commits on this package since 2020, its a branch of jsonlint which has has commits from 2021 but that is only for updating an abandoned dependency (nomnom).